resolwe_bio/processes/support_processors/bam_split.yml

```diff
@@ -126,11 +132,11 @@
       re-checkrc
       re-progress 0.4
 
-      samtools sort "${NAME}_${BUILD1}_unsorted.bam" "${NAME}_${BUILD1}"
+      samtools sort -o "${NAME}_${BUILD1}.bam" "${NAME}_${BUILD1}_unsorted.bam"
```

Comment:
You can put the `samtools` command changes into a separate commit. In that commit you can also change `$HEAD`, `$SELECTION`, and `$TAIL` to  `${HEAD}`, `${SELECTION}`, and `${TAIL}` throughout the code. And also change lines' 89-92 format from ``...`` to `$(...)`. From `NUM=`grep -v @ "${NAME}.sam" -m 1 -n | cut -d ':' -f 1`` to `NUM=$(grep -v @ "${NAME}.sam" -m 1 -n | cut -d ':' -f 1).